spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/hystrix/TraceCommand.java

```diff
@@ -33,6 +34,8 @@
  */
 public abstract class TraceCommand<R> extends HystrixCommand<R> {
 
+	private static final String HYSTRIX_PROTOCOL = "hystrix";
```

Comment:
Maybe protocol isn't the right name, since hystrix isn't a protocol. Module? Component?